src/test/kotlin/com/gildedrose/GildedRoseTest.kt

```diff
@@ -1,19 +1,168 @@
 package com.gildedrose
 
-import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.*
 import org.junit.jupiter.api.Test
 
 internal class GildedRoseTest {
 
     @Test
-    fun foo() {
-        val items = listOf(Item("foo", 0, 0))
+    fun normalUnexpiredItemDegrades() {
+        val items = listOf(
+            Item("Apples", 10, 50),
+            Item("Passes", 3, 20),
+            Item("Oranges", 1, 0)
+        )
         val app = GildedRose(items)
         app.updateQuality()
-        assertEquals("fixme", app.items[0].name)
+        val expected = listOf(
+            Item("Apples", 9, 49),
+            Item("Passes", 2, 19),
+            Item("Oranges", 0, 0)
```

Comment:
What happens if the code changes that stops an item going below 0 quality? This whole test would break, is that useful?